swagger/v1/component_schemas/ParticipantDeclarationAttributes.yml

```diff
@@ -43,7 +43,7 @@ properties:
       - npq-executive-leadership
     example: ecf-induction
   eligible_for_payment:
-    description: "The flag showing whether this participant eligible for funding"
+    description: "Indicates whether this participant would be eligible for payment from the DfE"
```

Comment:
I would say it indicates whether the _declaration_ is eligible for funding, not the _participant_.

Reply:
yep